.env.example

```diff
@@ -6,4 +6,4 @@ DEV_GITHUB_TOKEN=
 #GITHUB_SECRET=
 
 # Next-Auth secret (not required for local development)
```

Comment:
The description should be updated too as now it's actually required to make it work